spyder/plugins/completion/kite/widgets/calltoaction.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+from qtpy.QtCore import Qt, QTimer, QUrl
+from qtpy.QtGui import QDesktopServices
+from qtpy.QtWidgets import (QLabel, QApplication,
+                            QVBoxLayout, QFrame, QHBoxLayout,
+                            QPushButton)
+from spyder.utils.syntaxhighlighters import get_color_scheme
+
+from spyder.config.base import _
+from spyder.config.gui import get_font
+from spyder.config.manager import CONF
+from spyder.plugins.completion.kite.bloomfilter import KiteBloomFilter
+from spyder.plugins.completion.kite.parsing import find_returning_function_path
+from spyder.plugins.completion.kite.utils.status import check_if_kite_installed
+from spyder.plugins.completion.fallback.actor import FALLBACK_COMPLETION
+
+COVERAGE_MESSAGE = (
+    _("No completions found."
+      " Get completions for this case and more by installing Kite.")
+)
+
+
+class KiteCallToAction(QFrame):
+    def __init__(self, textedit, ancestor):
+        super(KiteCallToAction, self).__init__(ancestor)
+        self.textedit = textedit
+
+        self.setFrameStyle(54)
```

Comment:
Please change this to `self.setFrameStyle(QFrame.StyledPanel | QFrame.Plain)` (the equivalent using constants to the value `54`)